pkg/converter/constant.go

```diff
@@ -20,6 +20,9 @@ const (
 	LayerAnnotationNydusBootstrap     = "containerd.io/snapshot/nydus-bootstrap"
 	LayerAnnotationNydusSourceChainID = "containerd.io/snapshot/nydus-source-chainid"
 	LayerAnnotationNydusEncryptedBlob = "containerd.io/snapshot/nydus-encrypted-blob"
+	LayerAnnotationSourceDigest       = "containerd.io/snapshot/nydus-source-digest"
+	LayerAnnotationDestDigest 		  = "containerd.io/snapshot/nydus-dest-digest"
```

Comment:
How about renaming to `LayerAnnotationTargetDigest = "containerd.io/snapshot/nydus-target-digest"`.

Reply:
fixed